### ISSUE TITLE
Question: Newest Versions of mkvmerge

### ISSUE THREAD

**Issue by ulrick65** [binhex/videoslimmer#7]
It appears that somewhere along the line mkvmerge changed completely how it outputs the results of --identify.  I am assuming (but not sure) that they now want us to use mkvinfo instead.

At any rate, my regex sucks...so trying to decipher how to make your awesome videoslimmer work with newer versions of mkvmerge is beyond my reach at the moment.

Any thoughts on updating to a recent version perhaps?

Thanks.

**Comment by binhex** [comment]
yeah i will add it to the to do list, not sure when i will get around to it so for now try using an older version of mkvmerge until ive fixed it.

**Comment by ulrick65** [comment]
Thanks for adding it.  The newest version (starting in 2018) no longer returns the text information as it did before with the --identify.  Now it returns only a JSON payload...and you need to use the following switch:

--identification-format json --identify

Here's a link to the info on deprecating "--identify-verbose": 

[https://github.com/mbunkus/mkvtoolnix/commit/f35ba20dc33140daba218945a389acf2e23556e9](url)

**Comment by liquidsmoke66** [comment]
Anyone know which is the last version that works with videoslimmer?

**Comment by menos08642** [comment]
Any update to this?

**Comment by binhex** [comment]
i havent got around to looking at this yet, i know the code does need a fair bit of reworking, as im not completely happy with how its interacting with mkvmerge at present, i will try and make some time for it next week.

**Comment by binhex** [comment]
after a long time i have finally got around to giving videoslimmer some lovin, imminent release coming guys....

**Comment by gbougiakas** [comment]
This is amazing news. I checked the date of the last update of the app and downloaded the relevant mkv tools for it to work.

**Comment by RandomNinjaAtk** [comment]
Really looking forward to the update!

**Comment by binhex** [comment]
its coming guys, just got stuck on other stuff

**Comment by RandomNinjaAtk** [comment]
For everyone here still waiting, here is an alternate version that I was able to put together to perform similar functions:

https://github.com/RandomNinjaAtk/Scripts/blob/master/external/MKVCleanup.bash

**Comment by jpoll** [comment]
Hello @binhex. Do you still plan on updating this to work with newer versions of mkvtoolnix. Thank you for your work.

**Comment by binhex** [comment]
@jpoll yes, i have made progress on the next release but i keep getting side tracked onto other projects, but yes i do plan on releasing a new version, hopefully shortly.
